source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidSavedGameClient.cs

```diff
@@ -147,24 +147,23 @@ public AndroidSavedGameClient(AndroidJavaObject account)
                                                                 completedCallback,
                                                                 () => InternalOpen(filename, source, resolutionStrategy,
                                                                     prefetchDataOnConflict,
-                                                                    conflictCallback, completedCallback)
-                                                            );
+                                                                    conflictCallback, completedCallback));
 
                                 conflictCallback(resolver, original, null, unmerged, null);
                             }
-                        } else {
+                        }
+                        else
+                        {
                             using (var snapshot = dataOrConflict.Call<AndroidJavaObject>("getData"))
                             {
-                                completedCallback(SavedGameRequestStatus.Success, new AndroidSnapshotMetadata(snapshot.Call<AndroidJavaObject>("freeze")));
+                                AndroidJavaObject metadata = snapshot.Call<AndroidJavaObject>("freeze");
```

Comment:
using for AndroidJavaObject metadata = snapshot.Call<AndroidJavaObject>("freeze");